### ISSUE TITLE
performance of multiplying transposed matrix

### ISSUE THREAD

**Issue by lipiji** [mikiobraun/jblas#68]
Here is a problem which I am entangled with:
http://stackoverflow.com/questions/22288888/multiplying-transposed-matrix-with-jblas

Then any tips to optimize this?
Thanks.

**Comment by mikiobraun** [comment]
Yeah, BLAS supports multiplying in place, but I never exposed this to jblas. But one could have a custom function for multiplications of the form A^T B or AB^T
